app/views/reports/_review_report.html.erb

```diff
@@ -106,9 +106,14 @@
                     <!--later we can create a hyperlink question type to deal with this situation.-->
                     <%= list_hyperlink_submission(reviewer_map.id, 5386) if Assignment.find_by(id: @id.to_i).try(:course).try(:instructor).try(:name) == 'Jkidd' %>
                 <!-- Link that displays the time tracking detail -->
-                    <% (1..@assignment.num_review_rounds).each do |round| %>
-                      <a onclick="displayTimeDetail(<%= reviewer_map.id %>, <%= round %>)" href="javascript:void(0);"> <%= "R#{round}: "%><%= secondsToHuman(getTotalTime(reviewer_map.id, round)) %> </a>
-                    <% end %>
+                <% (1..@assignment.num_review_rounds).each do |round| %>
+                  <% show_time = secondsToHuman(getTotalTime(reviewer_map.id, round)) %>
+                  <% if show_time == "0 sec" %>
+                    <a style="pointer-events: none; color: gray;" disabled="disabled" href="javascript:void(0);"> <%= "R#{round}: No-data" %> </a>
```

Comment:
I think this change to what is displayed if there is no data makes more sense as it seems intentional. Having the hyphen there makes it seem as though something is not displaying correctly as opposed to something a user should see

```suggestion
                    <a style="pointer-events: none; color: gray;" disabled="disabled" href="javascript:void(0);"> <%= "R#{round}: No Data" %> </a>
```